app/react/Metadata/actions/actions.js

```diff
@@ -40,26 +40,26 @@ const resetMetadata = (metadata, template, options, previousTemplate) => {
   return resetedMetadata;
 };
 
-export function loadInReduxForm(form, onlyReadEntity, templates) {
+export function loadInReduxForm(form, _entity, templates) {
```

Comment:
@konzz works ok, but i think we should fix eslint warning on the code we touch at least, pls fix / refactor/fix lints on the action